.github/workflows/application-signals-python-e2e-ec2-test.yml

```diff
@@ -204,7 +204,8 @@ jobs:
           --log-group ${{ env.LOG_GROUP_NAME }}
           --service-name python-sample-application-${{ env.TESTING_ID }}
           --remote-service-name python-sample-remote-application-${{ env.TESTING_ID }}
-          --request-body ip=${{ env.REMOTE_SERVICE_IP }}
+          --remote-target-name ::s3:::e2e-test-bucket-name-${{ env.TESTING_ID }}
+          --request-body ip=${{ env.REMOTE_SERVICE_IP }}&testingId=${{ env.TESTING_ID }}
```

Comment:
Can we refactor this somehow? I like how you handled this but maybe spending some time on a slightly more clear/better solution to pass in the request body would be good for the future.

Reply:
Since the inputted value is actually a query-string, the current implementation is fairly explicit/straight-forward, although it would be more accurate to rename this as `"--query-string"` rather than `"--request-body"`. I can make this change.

If we were to make a refactor such that `ip=${{ env.REMOTE_SERVICE_IP }}` is only sent to `/remote-service` endpoint, and `testingId=${{ env.TESTING_ID }}` is only sent to `/aws-sdk-call` endpoint, we will need to make this association here and refactor the [`http-with-body` logic](https://github.com/aws-observability/aws-application-signals-test-framework/blob/0e48219507031be6e7f1c2e2a570d7944091da1c/validator/src/main/java/com/amazon/aoc/validators/ValidatorFactory.java#L67-L72) as well, which may introduce a refactor larger than the scope of this PR.